atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/metrics/TransactionalExpectationsMetrics.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.impl.metrics;
+
+import com.palantir.atlasdb.util.MetricsManager;
+
+public class TransactionalExpectationsMetrics {
```

Comment:
We have a plugin for generating metrics from yml. See `transactions4-knowledge.yml` for example